ansible/inventories/production/group_vars/inventory-next/vars.yml

```diff
@@ -31,13 +31,23 @@ inventory_ckan_bucket_name: "{{ inventory_next_ckan_s3_bucket_name }}"
 inventory_ckan_bucket_prefix: "{{ inventory_next_ckan_s3_bucket_prefix }}"
 ckan_site_domain: "{{ inventory_next_ckan_service_url }}"
 
-inventory_app_repo_branch: inventory_ckan_2.8
+inventory_app_repo_branch: inventory-28-with-saml2
 newrelic_app_name: inventory-next
 newrelic_enabled: true
 
+inventory_ckan_plugins_additional: [saml2auth s3filestore]
+
 # saml2 authentication
-inventory_ckan_saml2_enabled: false
-inventory_ckan_plugins_additional: [s3filestore]
+inventory_ckan_saml2_enabled: true
```

Comment:
I don't know if it's going to screw you up or not, but I'd like to disable login on production until [PIV card authentication is required](https://github.com/GSA/datagov-deploy/issues/2526). It's okay to leave it enabled for testing, just keep an eye on the accounts.

Reply:
saml2 enabled for inventory-next sandbox and staging. disabled for production.